src/main/java/Duke.java

```diff
@@ -1,10 +1,40 @@
+import Duke.Tool.*;
+import Duke.Task.*;
+
+/**
+ * Represent Duke, a task scheduling bot.
+ */
 public class Duke {
+    
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+    private Parser parser;
+    private Command command;
+
+    /**
+     * Construct a new Duke object.
```

Comment:
In Javadoc comments, there should be an empty line between description and parameter section.